kasa/tapo/tapodevice.py

```diff
@@ -104,11 +127,16 @@ def model(self) -> str:
     @property
     def alias(self) -> Optional[str]:
         """Returns the device alias or nickname."""
-        if self._info and (nickname := self._info.get("nickname")):
-            return base64.b64decode(nickname).decode()
-        else:
+        if not self._info:
             return None
 
+        if nickname := self._info.get("nickname"):
+            return base64.b64decode(nickname).decode()
+        elif self.is_strip:  # Fallback to model on empty nickname
+            return self._info.get("model")
```

Comment:
Maybe we should do this always, which would free the downstreams from checking for noneness?

Reply:
It might be better to let downstreams pick how they want to handle noneness because otherwise HA would show the model twice I think.

Could we add an Optional child_device_id to the TapoDevice so HA could do something like:
```
if not device.alias:
  if not device._child_device_id:
    alias = f"{device.parent.alias} ({ device._child_device_id })"
  alias = short_mac
```

Reply:
Although we may need to add some way to distinguish child devices from their parents, I think it's better to keep this change as simple as possible.

There was a PR comment about whether the alias should ever be none (which may be the case for discovered-but-not-updated devices), but I'm thinking whether we should remove the optional from it and just return an empty string (like it's for P300 main device) and leave it for downstreams to handle?

This way, no one would need to check for noneness but `if not alias`-type of checks would still allow making the decision on how to display it when none is available.